spec/models/profile_spec.rb

```diff
@@ -1035,14 +1035,15 @@
       expect(profile.deactivation_reason).to be_nil
       expect(profile.fraud_review_pending?).to eq(true) # changed
       expect(profile.gpo_verification_pending_at).to be_nil
-      expect(profile.in_person_verification_pending_at).to be_nil
+      expect(profile.in_person_verification_pending_at).to be_nil # changed
       expect(profile.initiating_service_provider).to be_nil
       expect(profile.verified_at).to be_nil
 
       expect(profile).to_not be_active
       expect(profile.fraud_review_pending?).to eq(true)
       expect(profile.fraud_rejection?).to eq(false)
       expect(profile.fraud_pending_reason).to eq('threatmetrix_review')
+      expect(profile.pending_reasons).to eq([:fraud_check_pending])
```

Comment:
Interesting aside: this is what was previously returning `[:in_person_verification_pending]` causing the problem. I set out to assert that this was an empty array, but it is not.

The `action-account` code handles this, and it's correct behavior, but it wasn't entirely intuitive in the moment.